test/powershell/Language/Parser/LanguageAndParser.TestFollowup.Tests.ps1

```diff
@@ -71,14 +71,14 @@ Describe "ScriptBlockAst.GetScriptBlock throws on error" -Tags "CI" {
 
     It "with parse error" {
         $ast = [System.Management.Automation.Language.Parser]::ParseInput('function ', [ref]$null, [ref]$e)
-        { $ast.GetScriptBlock() } | Should Throw
+        { $ast.GetScriptBlock() } | Should -Throw
```

Comment:
Update to check the errorid: `Should -Throw -ErrorId` for any cases that simply checks if an exception is thrown